flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/SingleOutputStreamOperator.java

```diff
@@ -416,4 +418,35 @@ private boolean canBeParallel() {
 		transformation.setSlotSharingGroup(slotSharingGroup);
 		return this;
 	}
+
+	/**
+	 * Gets the {@link DataStream} that contains the elements that are emitted from an operation
+	 * into the side output with the given {@link OutputTag}.
+	 *
+	 * <p>Example:
+	 * <pre>{@code
+	 * static final OutputTag<X> sideOutputTag = new OutputTag<X>("side-output") {};
+	 *
+	 * public void flatMap(X value, Collector<String> out) throws Exception {
```

Comment:
Comments seems out of date, I think we already decided to get ride of CollectorWrapper